Makefile

```
@@ -1,2 +1,16 @@
-badge.svg:
-	curl -o 'badge.svg' 'https://img.shields.io/badge/tested_with-Aqua.jl-05C3DD.svg?logo=data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8+CjwhLS0gQ3JlYXRlZCB3aXRoIElua3NjYXBlIChodHRwOi8vd3d3Lmlua3NjYXBlLm9yZy8pIC0tPgoKPHN2ZwogICB3aWR0aD0iMTkiCiAgIGhlaWdodD0iMTQiCiAgIHZpZXdCb3g9IjAgMCA1LjAyNzA4MzIgMy43MDQxNjY4IgogICB2ZXJzaW9uPSIxLjEiCiAgIGlkPSJzdmc1IgogICBzb2RpcG9kaTpkb2NuYW1lPSJiYWRnZS5zdmciCiAgIGlua3NjYXBlOnZlcnNpb249IjEuMi4yIChiMGE4NDg2NTQxLCAyMDIyLTEyLTAxKSIKICAgeG1sbnM6aW5rc2NhcGU9Imh0dHA6Ly93d3cuaW5rc2NhcGUub3JnL25hbWVzcGFjZXMvaW5rc2NhcGUiCiAgIHhtbG5zOnNvZGlwb2RpPSJodHRwOi8vc29kaXBvZGkuc291cmNlZm9yZ2UubmV0L0RURC9zb2RpcG9kaS0wLmR0ZCIKICAgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIgogICB4bWxuczpzdmc9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8c29kaXBvZGk6bmFtZWR2aWV3CiAgICAgaWQ9Im5hbWVkdmlldzEzIgogICAgIHBhZ2Vjb2xvcj0iI2ZmZmZmZiIKICAgICBib3JkZXJjb2xvcj0iIzAwMDAwMCIKICAgICBib3JkZXJvcGFjaXR5PSIwLjI1IgogICAgIGlua3NjYXBlOnNob3dwYWdlc2hhZG93PSIyIgogICAgIGlua3NjYXBlOnBhZ2VvcGFjaXR5PSIwLjAiCiAgICAgaW5rc2NhcGU6cGFnZWNoZWNrZXJib2FyZD0iMCIKICAgICBpbmtzY2FwZTpkZXNrY29sb3I9IiNkMWQxZDEiCiAgICAgc2hvd2dyaWQ9ImZhbHNlIgogICAgIGlua3NjYXBlOnpvb209IjI1LjI2MjExNiIKICAgICBpbmtzY2FwZTpjeD0iOC45MDY2MTcyIgogICAgIGlua3NjYXBlOmN5PSIyLjA3ODIxMDciCiAgICAgaW5rc2NhcGU6d2luZG93LXdpZHRoPSIxNDQ0IgogICAgIGlua3NjYXBlOndpbmRvdy1oZWlnaHQ9IjEyMzYiCiAgICAgaW5rc2NhcGU6d2luZG93LXg9IjQ1MyIKICAgICBpbmtzY2FwZTp3aW5kb3cteT0iMTcyIgogICAgIGlua3NjYXBlOndpbmRvdy1tYXhpbWl6ZWQ9IjAiCiAgICAgaW5rc2NhcGU6Y3VycmVudC1sYXllcj0ic3ZnNSIgLz4KICA8ZGVmcwogICAgIGlkPSJkZWZzMiIgLz4KICA8ZWxsaXBzZQogICAgIHN0eWxlPSJmaWxsOiMzODk4MjY7ZmlsbC1vcGFjaXR5OjE7c3Ryb2tlOiNmZmZmZmY7c3Ryb2tlLXdpZHRoOjAuMztzdHJva2UtbWl0ZXJsaW1pdDo0O3N0cm9rZS1kYXNoYXJyYXk6bm9uZTtzdHJva2Utb3BhY2l0eToxIgogICAgIGlkPSJwYXRoODU0IgogICAgIGN4PSIyLjUxODg3MTEiCiAgICAgY3k9IjEuNTIzODQxOSIKICAgICByeD0iMS4zMjI5MTYyIgogICAgIHJ5PSIxLjMyMjkxNjciIC8+CiAgPGNpcmNsZQogICAgIHN0eWxlPSJmaWxsOiNjYjNjMzM7ZmlsbC1vcGFjaXR5OjE7c3Ryb2tlOiNmZmZmZmY7c3Ryb2tlLXdpZHRoOjAuMztzdHJva2UtbWl0ZXJsaW1pdDo0O3N0cm9rZS1kYXNoYXJyYXk6bm9uZTtzdHJva2Utb3BhY2l0eToxIgogICAgIGlkPSJwYXRoMTAxMDMiCiAgICAgY3g9IjEuMTUzMjQ1OCIKICAgICBjeT0iMi41ODIxNzUiCiAgICAgcj0iMC45MjA4NTc3MyIgLz4KICA8Y2lyY2xlCiAgICAgc3R5bGU9ImZpbGw6Izk1NThiMjtmaWxsLW9wYWNpdHk6MTtzdHJva2U6I2ZlZmVmZTtzdHJva2Utd2lkdGg6MC4zO3N0cm9rZS1taXRlcmxpbWl0OjQ7c3Ryb2tlLWRhc2hhcnJheTpub25lO3N0cm9rZS1vcGFjaXR5OjEiCiAgICAgaWQ9InBhdGgxMDEwMy0zIgogICAgIGN4PSIzLjg4NDQ5NjUiCiAgICAgY3k9IjIuNTgyMTc1IgogICAgIHI9IjAuOTIwODU3NzMiIC8+CiAgPGcKICAgICBpZD0ibGF5ZXIyIgogICAgIHN0eWxlPSJkaXNwbGF5Om5vbmUiCiAgICAgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMC43OTM3NTA2MykiIC8+Cjwvc3ZnPgo='
+all: generate_badge generate_favicon
+
+.PHONY: generate_badge generate_favicon
```

```suggestion
.PHONY: all generate_badge generate_favicon
```
The target `all` is also phony because it does not actually produce a file named `all`. 

(Why declare target as phony? well, if there ever *is* a file `all` added by the user, then `make` or `make all` won't do anything anymore because `all` is already present.

---

Thank you for the comment! I have updated the file in https://github.com/JuliaTesting/Aqua.jl/pull/128/commits/d45445733e3776fc397a4a077c1c6c4d5bc7439a.